test/externalTests/zeppelin.sh

```diff
@@ -111,6 +112,9 @@ function zeppelin_test
     # Could result in an error in the future.
     sed -zi "s|it(\('deposit'\)|it.skip(\1|3" test/token/ERC20/extensions/ERC4626.test.js
     sed -zi "s|it(\('withdraw'\)|it.skip(\1|3" test/token/ERC20/extensions/ERC4626.test.js
+    # Avoid the testToInt(248) and testToInt(256) case
+    sed -i "s|range(8, 256, 8).forEach(bits => testToInt(bits));|  range(8, 240, 8).forEach(bits => testToInt(bits));|" test/utils/math/SafeCast.test.js
```

Comment:
The regex needs escaping:
```suggestion
    sed -i "s|range\(8, 256, 8\)\.forEach\(bits => testToInt\(bits\)\);|  range(8, 240, 8).forEach(bits => testToInt(bits));|" test/utils/math/SafeCast.test.js
```

Reply:
Also, you don't have to repeat the whole thing if you use match groups:
```suggestion
    sed -i "s|range\(8, 256, 8\)(\.forEach\(bits => testToInt\(bits\)\);)|range(8, 240, 8)\1|" test/utils/math/SafeCast.test.js
```
This could even be further reduced to just the `256` part but I think including the whole `range()` part makes it more readable.

Reply:
Looks like the escaping broke it. Only for match groups you need to escape...